axlearn/common/attention.py

```diff
@@ -2508,23 +2508,27 @@ def attention_thunk(target: Tensor) -> tuple[Optional[NestedTensor], Tensor]:
                 atten_state, atten_output = attention_thunk(TensorSpec(target.shape, target.dtype))
                 return dict(attention=atten_state), atten_output
 
+        remat_pt1 = "attention_output"
```

Comment:
Is this really needed? How much perf gain we get from saving `attention_output`?